Assets/Prefabs/Character/Character.prefab

```diff
@@ -231,7 +234,7 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   SyncPositionX: 1
-  SyncPositionY: 1
+  SyncPositionY: 0
```

Comment:
for tracking purposes, investigating the impact of cutting y axis

Reply:
No impact! On the server, PCs and NPCs are placed at a y position that is non-zero (0.0003771782) just due to nav mesh bounds of the player. Clients are spawned with 0 for height, and that difference is ultimately negligible to gameplay and look. If we had ramps, or more complex nav meshes, then we would need to sync y positions, but as the level is all flat, there is no need to sync y position.